packages/docs/package.json

```
@@ -28,22 +28,38 @@
     "qwik": "qwik"
   },
   "devDependencies": {
+    "@builder.io/mitosis": "latest",
```

this wont use the latest local version. is that on purpose?

```suggestion
    "@builder.io/mitosis": "workspace:*",
```